{"name": "test_create_event", "status": "broken", "statusDetails": {"message": "AttributeError: 'str' object has no attribute 'items'", "trace": "get_new_event_body = [{'customer_event_category_id': 331, 'name': 'Event80', 'reporting_end_time': '2023-09-23T09:56:40.499Z', 'reporting_start_time': '2023-08-23T09:56:40.499Z', ...}]\n\n    def test_create_event(get_new_event_body):\n    \n>       response = test.api_tests.conftest.events_api.create_events(body=get_new_event_body)\n\ntest_events_positive.py:11: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../api_service/events_api.py:35: in create_events\n    return self.api_executor.post(\n../../api_service/base_api.py:20: in post\n    return self.__execute_request('post', url=url, body=body, headers=headers)\n../../api_service/base_api.py:8: in __execute_request\n    return requests.request(\n/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/requests/api.py:59: in request\n    return session.request(method=method, url=url, **kwargs)\n/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/requests/sessions.py:575: in request\n    prep = self.prepare_request(req)\n/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/requests/sessions.py:486: in prepare_request\n    p.prepare(\n/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/requests/models.py:369: in prepare\n    self.prepare_headers(headers)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <PreparedRequest [POST]>\nheaders = 'eyJraWQiOiJzXC83Q3lJTWhIRGFHZVFWaFM4QjhKZVhGdmJTSElVUWVieWdQOGVUaGltWT0iLCJhbGciOiJSUzI1NiJ9.eyJjdXN0b206dHlwZSI6ImFk...JJy6tRoT7S3bLPUEezBCnRXkK5-SMbkDtk5ne-xZTSrMshJSc3dfjVxJoomWksvIo8nH830bN5JQCXdvdtVNgpppzClaADka9PzZ_Lo7QPkyZJUDkf90-g'\n\n    def prepare_headers(self, headers):\n        \"\"\"Prepares the given HTTP headers.\"\"\"\n    \n        self.headers = CaseInsensitiveDict()\n        if headers:\n>           for header in headers.items():\nE           AttributeError: 'str' object has no attribute 'items'\n\n/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/requests/models.py:489: AttributeError"}, "start": 1715116798579, "stop": 1715116798580, "uuid": "7ad49842-45e1-43db-8486-8fbca4fed998", "historyId": "a64b3df3a3f073c59efb071a07a66e31", "testCaseId": "a64b3df3a3f073c59efb071a07a66e31", "fullName": "test.api_tests.test_events_positive#test_create_event", "labels": [{"name": "parentSuite", "value": "test.api_tests"}, {"name": "suite", "value": "test_events_positive"}, {"name": "host", "value": "dariias-mbp.home"}, {"name": "thread", "value": "81367-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.api_tests.test_events_positive"}]}